### ISSUE TITLE
Prettier graphs

### ISSUE THREAD

**Issue by zevv** [zevv/npeg#13]
I'd love to be able to draw graphs like this from a grammar:

![Graph](https://zevv.nl/div/.old/sdexample.gif)

Or ascii:

```
                ╭─(back)◂─╮
  ──▸(term)─╭───╰─────────╯──╮──▸
            ╰─(one)◂─╮─(+)◂──╯
                     ╰─(-)◂──╯
```

**Comment by zevv** [comment]
That should work, this is rendered from a grammar:

```
  ╭────────────────────»────────────────────╮                        
  │ ╭────»────╮                             │   ╭────────»────────╮  
o─┴┬┴┬─"aap"─┬┴─»──'('──»─┬─Digit─┬─»──')'─┬┴─»─┴─flop──»─┬─Word─┬┴─o
   │ ╰───«───╯            ╰───«───╯        │              ╰──«───╯   
   │                 ╭─»─╮                 │                         
   ├─────────────────┴─1─┴─────────────────┤                         
   │               ╭───»───╮               │                         
   ╰───────────────┴┬─bla─┬┴───────────────╯                         
                    ╰──«──╯          
```

**Comment by zevv** [comment]
Closed by 599af1b0daecd71fa9c2772745f81a37bef14d48
